packages/web/package.json

```diff
@@ -41,7 +41,7 @@
   "dependencies": {
     "@apollo/client": "^3.3.12",
     "@apollo/react-ssr": "^4.0.0",
-    "@journaly/j-db-client": "^10.0.2",
+    "@journaly/j-db-client": "^11.0.0",
```

Comment:
Oh, this reminds me, we also need to bump this version number in j-mail and remember to deploy the change

Reply:
Excellent catch!